x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -351,9 +352,13 @@ export const pushCase = async (
 };
 
 export const getActionLicense = async (signal: AbortSignal): Promise<ActionLicense[]> => {
-  const response = await KibanaServices.get().http.fetch<ActionLicense[]>(ACTION_TYPES_URL, {
-    method: 'GET',
-    signal,
-  });
-  return response;
+  const response = await KibanaServices.get().http.fetch<ActionLicense[]>(
+    getAllConnectorTypesUrl(),
+    {
+      method: 'GET',
+      signal,
+    }
+  );
+
+  return convertArrayToCamelCase(response) as ActionLicense[];
```

Comment:
Just curious why we need `convertArrayToCamelCase` now? Did the response from the action API change?

Reply:
@jonathan-buttner Yes, the response has changed and all properties are in snake case. Example: https://www.elastic.co/guide/en/kibana/current/list-connector-types-api.html#list-connector-types-api-example